jib-core/src/main/java/com/google/cloud/tools/jib/api/LayerConfiguration.java

```diff
@@ -167,7 +184,10 @@ public Builder addEntryRecursive(
         BiFunction<Path, AbsoluteUnixPath, FilePermissions> filePermissionProvider)
         throws IOException {
       return addEntryRecursive(
-          sourceFile, pathInContainer, filePermissionProvider, DEFAULT_MODIFIED_TIME_PROVIDER);
+          sourceFile,
+          pathInContainer,
+          DEFAULT_FILE_PERMISSIONS_PROVIDER,
```

Comment:
Shouldn't this be `filePermissionProvider`?

Reply:
Gosh, nice catch. Glad you got this.